### DISCUSSION TITLE
Help needed choosing theme for non-blog site

### DISCUSSION THREAD

**Discussion by PeteBleackley** [getpelican/pelican#3122] [Q&A]
I'm planning on setting up a portfolio site. It won't be a blog, but it will have multiple pages. I'm looking for a suitable theme to use for it, almost all the theme's I've seen are blog-oriented. The only exception I've found so far is resume, but that seems to be intended for single page sites. Can anybody suggest something suitable?

**Comment by datatalking** [comment]
@PeteBleackley I'm doing the exact same thing, but I can't seem to find a gallery of Pelican samples to choose from.

What I did find was the HUGO framework had several I liked and since I've posted about how the pelicantheme webpage is down a few that I liked on Hugo are. https://themes.gohugo.io/themes/mainroad/ or https://themes.gohugo.io/themes/congo/.

**Comment by justinmayer** [comment]
Most themes, for Pelican or otherwise, are focused on chronological post content. Indeed, both of the themes suggested by @datatalking are centered around chronological post content and unlikely to be representative of the kind of site that @PeteBleackley is looking for: multiple pages centered around a portfolio.

I imagine such Pelican themes exist, but off the top of my head I can't tell you which those are. It would require some digging around. While waiting for a more comprehensive Pelican theme gallery to be implemented in an official way, you can use the following **very-much-temporary-and-not-a-long-term-solution** site: https://avaris.github.io/pelican-themes/
